website/src/content/docs/docs/handbook/breaking-change-policy.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Breaking Change Policy
+---
+
+# Breaking Change Policy
+
+import { Badge } from "@astrojs/starlight/components";
+
+:::note
+The information in this document is applicable to TypeSpec packages with version 1.0.0 or greater.
+:::
+
+## Versioning Scheme
+
+The TypeSpec Compiler and libraries follow a major, minor, patch versioning scheme according to the following policy:
+
+## Breaking Change Philosophy
+
+Existing language syntax and semantics will not change without a major version.
+The runtime behavior for a given TypeSpec will not change without a major version.
+TypeSpecs leveraging new or updated features may cause runtime API consumers to fail
+(e.g. because a new type is unhandled in an emitter or library that it uses)
+or builds to fail (e.g. because exhaustive unions are no longer exhaustive).
+
+- The language syntax and semantics for existing language elements will not change within a major version.
+- A spec that builds successfully with a version of the TypeSpec compiler and its libraries will build successfully with a newer version of the compiler and libraries within the same major version.
+- New types and functionality may be added in a minor version release to the compiler or any TypeSpec library.
+  - If a spec uses the new type or functionality, existing libraries or emitters may not work with it, and this is not considered a breaking change in the compiler or library that introduced the type. Spec authors should expect that _existing_ specs will continue to work with their workflows, but not that newly-added features will work with their existing workflows every time.
+- TypeScript API types in TypeScript might change in ways that introduce TypeScript type checking errors (e.g. adding a new variant to a union). TypeScript type checking errors in library builds are not considered breaking changes, and library authors working in TypeScript should take care to program defensively as appropriate. More information about TypeScript API guarantees is provided in the section _Categories of Breaking Changes_ below.
+
+### Bugs
+
+A bug fix that introduces a technically breaking change will not be considered a breaking change for the purposes of TypeSpec versioning if it is clear that the existing behavior was broken or unintended in the first place.
+
+## Categories of Breaking Changes
+
+- <Badge text="Tier 0" variant="success" />: These are stable core APIs and language features that
+  define fundamental semantics and will not be broken without a major version update. We recommend
+  that library authors depend on these APIs.
+- <Badge text="Tier 1" variant="caution" />: These are APIs that have very limited use cases and may
+  be broken without a major version update. We recommend that library authors avoid depending on
+  these APIs.
+- <Badge text="Tier 3" variant="danger" />: These are internal APIs intended only for use by the
```

Comment:
Probably we want this to be tier 2 and the tier below to be tier 3.  (Later in this paragraph, this tier is described as tier 2)